.dockerignore

```diff
@@ -0,0 +1,6 @@
+.dockerignore
+Dockerfile
+docker-compose.yml
```

Comment:
Top-level files should be minimum. Move docker related files to `resources/docker`?